packages/devtools_app/lib/src/table.dart

```diff
@@ -209,16 +209,17 @@ class SelectionNotifier<T> extends ValueNotifier<T> {
 /// this case, tree table operations (expand all, collapse all, sort, etc.) will
 /// be applied to every tree.
 class TreeTable<T extends TreeNode<T>> extends StatefulWidget {
-  TreeTable({
-    Key key,
-    @required this.columns,
-    @required this.treeColumn,
-    @required this.dataRoots,
-    @required this.keyFactory,
-    @required this.sortColumn,
-    @required this.sortDirection,
-    this.selectionNotifier,
-  })  : assert(columns.contains(treeColumn)),
+  TreeTable(
+      {Key key,
+      @required this.columns,
+      @required this.treeColumn,
+      @required this.dataRoots,
+      @required this.keyFactory,
+      @required this.sortColumn,
+      @required this.sortDirection,
+      this.selectionNotifier,
+      this.autoExpandRoot})
```

Comment:
trailing comma, and rename to autoExpandRoots

Reply:
Done